data/data_store.py

```diff
@@ -148,18 +148,16 @@ def _build_data_for_symbol(self, symbol: str, data_type: DataType):
 
         path = data_info.get_path(symbol)
 
-        if not _check_file(path):
-            data = data_info.get_data(symbol)
-            write_csv(path, data, data_info.fields)
```

Comment:
Why is this check gone?
This way, every time the store gets built, every API call is made again, no matter if the store is already filled or not. Build should just ensure, that every file in the new configuration (stocks, date range) is available. 
You would have to comment out the build() data store API call every time you run a script the second time to make use of the caching. 

Or am I missing something?